components/brave_new_tab_ui/components/default/bitcoinDotCom/index.tsx

```diff
@@ -0,0 +1,305 @@
+}
+
+interface Props {
+  showContent: boolean
+  stackPosition: number
+  onShowContent: () => void
+  onBuyCrypto: () => void
+  onInteraction: () => void
+}
+
+class BitcoinDotCom extends React.PureComponent<Props, State> {
+  private assets: Record<string, string>
+  private fiatCurrencies: Record<string, string>
+
+  constructor (props: Props) {
+    super(props)
+    this.state = {
+      selectedAsset: 'BCH',
+      selectedFiat: 'EUR',
+      currentAmount: '',
+      assetsShowing: false,
+      fiatCurrenciesShowing: false
+    }
+    this.assets = {
+      'BCH': 'Bitcoin Cash',
+      'BTC': 'Bitcoin',
+      'ETH': 'Ethereum',
+      'LTC': 'Litecoin',
+      'XLM': 'Stellar',
+      'XRP': 'Ripple'
+    }
+    this.fiatCurrencies = fiatData
+  }
+
+  renderTitle () {
+    const { showContent } = this.props
+
+    return (
+      <Styled.Header>
+        <Styled.Title contentShowing={showContent}>
+          <Styled.BitcoinDotComIcon>
+            <img src={BitcoinDotComLogo} />
+          </Styled.BitcoinDotComIcon>
+          <Styled.TitleText>
+            {'Bitcoin.com'}
+          </Styled.TitleText>
+        </Styled.Title>
+      </Styled.Header>
+    )
+  }
+
+  toggleAssetsShowing = () => {
+    this.setState({
+      assetsShowing: !this.state.assetsShowing
+    })
+    this.props.onInteraction()
+  }
+
+  setSelectedAsset = (asset: string) => {
+    this.setState({
+      assetsShowing: false,
+      selectedAsset: asset
+    })
+  }
+
+  setCurrentAmount = ({ target }: any) => {
+    this.setState({
+      currentAmount: target.value
+    })
+    this.props.onInteraction()
+  }
+
+  toggleFiatCurrenciesShowing = () => {
+    this.setState({
+      fiatCurrenciesShowing: !this.state.fiatCurrenciesShowing
+    })
+    this.props.onInteraction()
+  }
+
+  setSelectedFiat = (fiat: string) => {
+    this.setState({
+      fiatCurrenciesShowing: false,
+      selectedFiat: fiat
+    })
+  }
+
+  hideFiatCurrencyOptions = () => {
+    this.setState({
+      fiatCurrenciesShowing: false
+    })
+  }
+
+  openBuyURL = () => {
+    const {
+      currentAmount,
+      selectedAsset,
+      selectedFiat
+    } = this.state
+
+    const amount = currentAmount.trim()
+    const asset = selectedAsset.toLowerCase()
+    const fiat = selectedFiat.toLowerCase()
+
+    if (amount.length) {
+      window.open(`https://bitcoincom.moonpay.io/?currencyCode=${asset}&baseCurrencyCode=${fiat}&baseCurrencyAmount=${amount}`, '_blank', 'noopener')
+    }
+
+    this.props.onBuyCrypto()
+  }
+
+  openInfoURL = () => {
+    window.open('https://bitcoincom.moonpay.io/', '_blank', 'noopener')
+    this.props.onInteraction()
+  }
+
+  renderCurrencyInput () {
+    const { assetsShowing, selectedAsset } = this.state
+
+    return (
+      <>
+        <Styled.InputHeader>
+          <Styled.InputLabel>
+            {getLocale('bitcoinDotComWidgetCurrency')}
+          </Styled.InputLabel>
+        </Styled.InputHeader>
+        <Styled.Dropdown
```

Comment:
This is not useable without a mouse. It doesn't allow focus, doesn't allow moving between options, and doesn't allow selecting an option.
It is also otherwise not accessible as it doesn't describe what is a label and what is options. Screen readers will just read the whole thing out before moving past.
If you're providing that in a follow-up, I suggest we start integrating using primitives from https://react-spectrum.adobe.com/react-aria/useSelect.html

Reply:
I would like to do accessibility followups as part of the upcoming refactoring sprint

Reply:
Might need an accessibility sprint!